circle.yml

```diff
@@ -37,12 +37,13 @@ workflows:
             - static-analysis
           filters:
             branches:
-              ignore: /^(master|release-.*)/
+              ignore: /^(main|release-.*)/
       - release-snapshot:
           filters:
             branches:
               only:
-                - master
+                - main
+                - release-1.x
```

Comment:
This is the only line that has to be added, all other changes from the PR are not necessary to publish a snapshot.

Reply:
yeah, but I renamed everything missed(I believe)